views/partials/letter.handlebars

```diff
@@ -4,8 +4,8 @@
 
 <p>The primary source of funding for HackSI is from corporate sponsors, such as yourself. HackSI receives no governmental funds.</p>
 
-<p>This exciting event will be held November 7th through November 8th at Dunn-Richmond center in Carbondale.
-It is the first ever Southern Illinois Hackathon and we look forward to providing this to the geeks of Southern Illinois.</p>
+<p>This exciting event will be held November 5th through November 6th at tne Dunn-Richmond Center in Carbondale.
+It is the 4th annual Southern Illinois Hackathon and we look forward to providing this to the geeks of Southern Illinois.</p>
```

Comment:
We missed these dates. They probably should be variables that we control from a config..